src/keys/secret.rs

```diff
@@ -0,0 +1,112 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use super::public::PublicKey;
+use super::stealth::StealthAddress;
+use crate::{permutation, ViewKey};
```

Comment:
and merge with this :)

Reply:
I now see that this is also happening in other places

Reply:
thank you, will check it out